app/views/spree/products/_cart_form.html.erb

```diff
@@ -1,83 +1,165 @@
+        <%# used_variants_options(@variants, @product).each_with_index do |option_type, index| %>
           <li>
-            <% if color_option_type_name.present? && color_option_type_name == option_type[:name] %>
-              <%= render "color_option_type", option_type: option_type, index: index %>
-            <% else %>
-              <%= render "option_type", option_type: option_type, index: index %>
-            <% end %>
+            <%# if color_option_type_name.present? && color_option_type_name == option_type[:name] %>
+              <%#= render "color_option_type", option_type: option_type, index: index %>
+            <%# else %>
+              <%#= render "option_type", option_type: option_type, index: index %>
+            <%# end %>
           </li>
-        <% end %>
+        <%# end %>
       </ul>
-    <% end %>
+    <%# end %>
 
-    <% if is_product_available_in_currency && @product.can_supply? %>
-      <%= hidden_field_tag "variant_id", default_variant.id %>
+    <%# if is_product_available_in_currency && @product.can_supply? %>
+      <%#= hidden_field_tag "variant_id", default_variant.id %>
 
       <hr>
       <div>
         <div class="mb-5 mt-4 text-center text-md-left">
-          <%= render 'spree/shared/quantity_select', input_name: :quantity %>
+          <%#= render 'spree/shared/quantity_select', input_name: :quantity %>
         </div>
 
-        <%= button_tag class: 'btn btn-primary w-100 text-uppercase font-weight-bold add-to-cart-button', id: 'add-to-cart-button', type: :submit, disabled: true do %>
-          <%= Spree.t(:add_to_cart) %>
-        <% end %>
+        <%#= button_tag class: 'btn btn-primary w-100 text-uppercase font-weight-bold add-to-cart-button', id: 'add-to-cart-button', type: :submit, disabled: true do %>
+          <%#= Spree.t(:add_to_cart) %>
+        <%# end %>
       </div>
-    <% end %>
+    <%# end %>
+  </div> -->
+<%# end %>
+<% is_product_available_in_currency = product_available_in_currency? %>
+<% default_variant = default_variant(@variants, @product) %>
+<div class="product-side-details d-flex flex-column justify-content-between h-100"  >
+  <div class="top-detail" >
+    <%= form_for :order, html: {
+      id: 'add-to-cart-form',
+      class: 'add-to-cart-form',
+      'data-product-summary': @product_summary.to_json,
+      'data-variants': product_variants_matrix(is_product_available_in_currency),
+      'data-variant-change-trigger-identifier': variant_change_identifier
+      } do |f| %>
+        <div class="product-options-header d-flex align-items-center justify-content-between" id="inside-product-cart-form" data-hook="inside_product_cart_form">
+          <div class="left">Choose your <strong>option:</strong></div>
+          <div class="right">
+            <div class="text-center text-md-left text-uppercase " id="product_variant_sku">
+                <%= default_variant.sku  %>
```

Comment:
@hamzanaveed9 , This section should show the sku of selected variant and show sku of default variant incase no variant selected. Please copy exact logic from BEST. Its implemented there.

Reply:
sir we have done this using js class it is selecting default variant and it changes when we select some different variant accordingly